Update copy: offerings

timcasasola opened the pull request:
Updated copy to speak directly to client needs ("You want X. We'll do Y.").

timcasasola committed:
Updated offerings copy

Updated copy to speak directly to client needs ("You want X. We'll do Y.").

devinhalladay commented on views/elements/content/right.md:
```diff
@@ -4,10 +4,10 @@
 ## Augment your team
-We'll embed ourselves within your company to solve tough problems, and move your product forward, alongside your entire team.
\ No newline at end of file
+You need an interim product design team. We'll embed ourselves within your company to solve tough problems.
```
I'd say "interim" makes us sound like middlemen…can we change this to something like:

You need an in-house product design team, or to add new minds to your existing team. We'll embed ourselves within your company to solve tough problems.

Not previous about that copy, but I think making it clear that we can either augment your existing team or work with you as your in-house product team is important

timcasasola replied:
Word, sounds good to me!

devinhalladay reviewed (changes requested):
@timcasasola just requested one change on the team augmentation section to make it sound more powerful and add that we can 1) augment an existing team or 2) be your first design team

timcasasola committed:
Update embedded teaming copy

Making clear that clients can
a) hire us as a "new" product team
b) hire us as new members to their existing team

timcasasola commented:
@devinhalladay copy changes made per your feedback! Up to merge?

devinhalladay reviewed (approved):
lgtm, great work
